src/helpers/gpt.ts

```diff
@@ -0,0 +1,274 @@
+import { getBotConfig, getBotContext, getLogger } from "../bindings";
+import { Payload, StreamlinedComment, UserType } from "../types";
+import { getAllIssueComments, getAllLinkedIssuesAndPullsInBody, getIssueByNumber } from "../helpers";
+import OpenAI from "openai";
+import { CreateChatCompletionRequestMessage } from "openai/resources/chat";
+import { ErrorDiff } from "../utils/helpers";
+import { Context } from "probot";
+
+export const sysMsg = `You are an AI designed to provide accurate technical answers. \n
+Whenever appropriate, format your response using GitHub Flavored Markdown. Utilize tables, lists, and code blocks for clear and organized answers. \n
+Do not make up answers. If you are unsure, say so. \n
+Original Context exists only to provide you with additional information to the current question, use it to formulate answers. \n
+Infer the context of the question from the Original Context using your best judgement. \n
+All replies MUST end with "\n\n <!--- { 'OpenAI': 'answer' } ---> ".\n
+`;
+
+export const specCheckTemplate = `
+Using the provided context, ensure you clearly understand the specification of the issue.
+If you do not understand the specification, infer it from the changes made.
+
+You are the first line of defense to ensure the specification is met before reviewers do their job.
+If the specification is not met, you are to provide a clear and concise reason why.
+
+Your response will be posted as a GitHub comment for everyone see so try to compliment the GitHub UI.
```

Comment:
```suggestion
Your response will be posted as a GitHub comment for everyone see in the pull request review conversation. 
```

Perhaps this is why it was laying on the compliments so thick.

Reply:
hahahaha aye very good 🤣

Reply:
```suggestion
Your response will be posted as a GitHub comment for everyone to see in the pull request review conversation. 
```